.travis.yml

```diff
@@ -1,15 +1,21 @@
-sudo: true
 language: go
-go:
-  - 1.9
-dist: trusty
+sudo: required
 services:
   - docker
-install:
-  - sudo apt-get install -y build-essential swig3.0 oracle-java8-installer g++ python3-dev
-  - go get -t -v ./...
-script: 
+go:
+  - "1.10"
+
+before_install:
+  - sudo sysctl -w vm.max_map_count=262144
+  - docker pull kuzzleio/sdk-cross:amd64
+  - go get -u ./...
+
+script:
   - ./test.sh
-  - docker run --rm -it --mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:amd64 /build.sh
+  - docker run -a stdout -a stderr --net host --rm --privileged --name build-machine --mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:amd64 /build.sh
+  - sh .codepipeline/start_kuzzle.sh
+  - docker run -a stdout -a stderr --privileged --rm --network codepipeline_default --link kuzzle --name test-machine --mount type=bind,source="$(pwd)",target=/go/src/github.com/kuzzleio/sdk-go kuzzleio/sdk-cross:amd64 /test.sh
+  - docker kill codepipeline_elasticsearch_1 codepipeline_redis_1 kuzzle
```

Comment:
Why do we bother killing the containers?

Reply:
Just a habit :smiley: